Selenium_ProjectMars/Tests/EducationTests.cs

```diff
@@ -0,0 +1,249 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using Selenium_ProjectMars.Models;
+using Selenium_ProjectMars.Pages;
+using Selenium_ProjectMars.Utilities;
+using SeleniumProjectMars.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_ProjectMars.Tests
+{
+    [TestFixture]
+    public class EducationTests
+    {
+        private BaseTest baseTest;
+        private IWebDriver driver;
+        private LoginPage loginPage;
+        private BasePage basePage;
+        private EducationPage educationPage;
+        protected AppConfigModel config;
+
+        private List<LoginModel> loginData;
+        private List<EducationModel> educationData;
+        private List<UpdatedEducationModel> updatedEducationData;
+
+        private string expectedMessage;
+        private string displayedMessage;
+
+        int addCount = 0;
+
+        [OneTimeSetUp]
+        public void GlobalSetup()
```

Comment:
you can move the one time setup in a common place as it will happen once for all tests